Missing "fastboot" dependency

```
Cannot find module 'fastboot'
Error: Cannot find module 'fastboot'
    at Function.Module._resolveFilename (module.js:325:15)
    at Function.Module._load (module.js:276:25)
    at Module.require (module.js:353:17)
    at require (internal/module.js:12:17)
    at Object.<anonymous> (/home/travis/build/simplabs/ember-cookies/node_modules/ember-cli-fastboot/index.js:10:16)
    at Module._compile (module.js:409:26)
    at Object.Module._extensions..js (module.js:416:10)
    at Module.load (module.js:343:32)
    at Function.Module._load (module.js:300:12)
    at Module.require (module.js:353:17)
    at require (internal/module.js:12:17)
```

it looks like we are using the `fastboot` package [here](https://github.com/ember-fastboot/ember-cli-fastboot/blob/c03aa4c24ba6dacc1dda8aca72d21d8ccfcce94c/index.js#L10) but it is not declared as a dependency in the [`package.json`](https://github.com/ember-fastboot/ember-cli-fastboot/blob/c03aa4c24ba6dacc1dda8aca72d21d8ccfcce94c/package.json#L21-L42) file

--- a/package.json
+++ b/package.json
@@ -31,6 +31,7 @@
     "ember-cli-eslint": "^3.0.2",
     "ember-cli-version-checker": "^1.3.1",
     "express": "^4.8.5",
+    "fastboot": "1.0.0-rc.6",
     "fastboot-express-middleware": "1.0.0-rc.7",
     "fastboot-filter-initializers": "0.0.2",
     "lodash.defaults": "^4.0.1",

